pkg/controller/controller.go

```diff
@@ -676,6 +683,8 @@ func (cont *AciController) enableGlobalSync(stopCh <-chan struct{}) {
 				eq := reflect.DeepEqual(expectedMap, policyNames)
 				if !eq {
 					cont.log.Debug("Wrong snatpolicy entry in snatglobalinfo for node: ", nodeName)
+					cont.log.Debug("Snatpolicy list according to snatglobalinfo: ", expected[nodeName])
+					cont.log.Debug("Snatpolicy list according to nodeinfo: ", policyNames)
```

Comment:
Adding debug logs here unfortunately does not achieve the purpose of providing the Info level instrumentation that I was requesting after L692. If the logging level is Info we will not see the above. I do understand that these variables are probably out of scope at L692, but there should be a way of expanding the scope.

Reply:
Then I will make the logs from L666-668 and the other two similar logs to info mode, anyway they would only print when there's a discrepancy. Does that sound good?

Reply:
Yeah, if that will get logged only in the case of discrepancy (meaning when the current info level log will get logged), then it definitely serves the purpose.